chapter3/src/3_4_1/main.rs

```diff
@@ -0,0 +1,29 @@
+use std::io::{stdin, Error, ErrorKind, Read};
+
+fn main() {
```

Comment:
main 関数を下記のようにしておいて、

```suggestion
fn main() -> std::io::Result<()> {
```